### DISCUSSION TITLE
Challenge #3: Planning for New Housing Development

### DISCUSSION THREAD

**Discussion by jacobrharris** [datakind/datakit-housing-fall-2024#3] [Challenges]
# Planning for New Housing Development

Availability and accessibility of affordable housing is a national problem. Public and private partnerships are looking to close this housing gap through construction and they need access to information and tools that would help them support getting new housing in the market as efficiently as possible. In this challenge, you’ll work to generate insights, repeatable processes, and prototype tools for housing stakeholders to use to inform their community action. 

* The big question a housing stakeholder is trying to answer with the results of this challenge is: where are economic incentives available to support housing development and what insights can be drawn about the communities where incentives are / are not available?  
* The big question a community resident is trying to answer is: are there incentives available that could enable me to rent or own in my target community?

In this challenge, you’ll help create an understanding of the housing market and risks to affordable housing loss and opportunities for new construction. 

## Community focus: Florida

One key actor in the affordable housing funding landscape is the [Florida Community Loan Fund](https://fclf.org) (FCLF). FCLF is a leader in financing affordable housing construction in the state and collaborator of community development financial institutions (CDFIs) across the Southeast region. As a CDFI, FCLF maximizes opportunities for people and places outside of the economic mainstream. One program that FCLF manages is a grant fund of the federal [Capital Magnet Fund](https://www.occ.gov/publications-and-resources/publications/community-affairs/community-developments-fact-sheets/pub-fact-sheet-capital-magnet-fund.pdf) which helps fund affordable housing in Florida. DataKind and FCLF explored Capital Magnet Fund tooling during a Fall 2023 DataDive collaboration. Stay tuned for updates on housing development needs in Florida after this DataKit event.

## Get started with existing data

[Grab the data here](https://github.com/datakind/datakit-housing-fall-2024/releases/latest).

Create an understanding of where affordable housing development is likely to be funded or otherwise incentivized. There are several federal government supported programs that offer financial support for affordable housing construction. Starting with three programs, the HUD [Low Income Housing Tax Credit (LIHTC) program](https://www.huduser.gov/portal/datasets/qct.html) Qualified Census Tracts, the HUD [Opportunity Zones Program](https://opportunityzones.hud.gov) Census Tracts, and the Dept of Treasury [CDFI Fund Investment Areas](https://home.treasury.gov/policy-issues/small-business-programs/state-small-business-credit-initiative-ssbci/2021-ssbci/cdfi-fund-investment-areas). Data from these three programs is contained within the EODatascape. Using this data, create an understanding of housing construction opportunities for financial support at a Census tract level. 

* Create an understanding of initial funding distributions \- where CDFI lending is reported and the number of mortgages originated at a Census tract level using EODatascape variables Total CDFI Lending Reported (CDFI Fund Transaction Level Report) and Number of Mortgages Originated (HMDA LAR).   
* Explore these housing development variables alongside population economic and demographic variables to create an understanding of populations and housing development opportunities.   
* Explore housing development variables alongside environmental variables from the EODataScape such as those sourced from the EPA Environmental Justice program: traffic proximity, hazardous waste proximity, air toxics cancer risk and the First Street Foundation: risk of flood, risk of fire. 

DataKind has [pulled the data](https://github.com/datakind/datakit-housing-fall-2024/releases/latest) already for two case study states: California and Florida, and we encourage you to explore additional geographies as well through the [EODatascape](https://eodatascape.datakind.org/).

This “getting started” analysis should help answer the following questions:

* Do these the geographies for these three programs overlap?  
* Do the geographies align to high population density?   
* Does the population breakdown shift by geography?  
* Are the areas of incentive in areas of potential environmental hazard?   
* Analysis insights and questions: What surprised you from this analysis? What are some limitations of the analysis? What are ways to extend the work?

## Take it further 

* Research: how are other communities assessing construction needs, and what solutions have been proposed to meet those needs? What datasets are available to answer these questions? This might include zoning data, conversion data (e.g. from commercial to residential properties), and more.   
* Data analysis and visualization:  
  * Are the areas where funding programs incentivize building located in connected communities? E.g. is there transit, walkability, infrastructure. Data from the [Smart Location](https://www.epa.gov/smartgrowth/smart-location-mapping) and AllTransit projects could be a helpful place to start. Accessible and affordable housing should be have access to jobs and workers, explore housing and [Access to Jobs and Workers Via Transit Tool (EPA)](https://www.epa.gov/smartgrowth/smart-location-mapping)  
  * What percentage of LIHTC properties are set to expire in the next 1,3,5 years? What percentage of housing stock do they represent? Where are those expirations located geographically?  
  * Explore the [National Housing Preservation Database](https://preservationdatabase.org/), generating summary statistics and maps pertaining to low-income housing.   
    * How does your target geography compare to the rest of the region? How does it compare to the rest of the United States as a whole?   
    * Does your target geography reserve more units for certain protected groups compared to other locations?   
    * Are units in your target geography more likely to have received violations upon inspection?   
    * How does the proportion of units in each building reserved for low-income housing compare to other locations?   
    * Does fair market rent have any relation to the number of units available?  
* Prototype:  
  * What type of tool can be prototyped to support urban planners in defining what the built environment in these geographies needs?  
  * What might the design of a data hub to track ownership and affordability commitments of housing units, ensuring transparency and accountability in affordable housing developments look like?

## Additional data sets to explore

* [National Housing Preservation Database](https://preservationdatabase.org/)  
  * [Smart Location Mapping | US EPA](https://www.epa.gov/smartgrowth/smart-location-mapping)  
  * [AllTransit](https://alltransit.cnt.org)  
  * [CDC/ATSDR SVI: Data and Documentation Download | Place and Health](https://www.atsdr.cdc.gov/placeandhealth/svi/data_documentation_download.html)  
  * [Low-Income Housing Tax Credit (LIHTC): Property Level Data | HUD USER](https://www.huduser.gov/portal/datasets/lihtc/property.html)  
  * [USHMC-Housing-Supply-source | HUD USER](https://www.huduser.gov/portal/ushmc/hs_source.html)

**Comment by MarlieKJ** [comment]
Hello, Just getting started looking at the California data. I am looking at the data dictionary, which defines the columns. 
I do not see a definition of "geoid", either in the provided data dictionary or at https://eodatascape.datakind.org/data-dictionary
Where can I find more information on the meaning of "geoid"? Is this a label for a piece of land that is used commonly?
Thank you!
